Doc/library/asyncio.rst

```diff
@@ -56,9 +56,13 @@ Additionally, there are **low-level** APIs for
 * :ref:`bridge <asyncio-futures>` callback-based libraries and code
   with async/await syntax.
 
+.. include:: ../includes/wasm-notavail.rst
```

Comment:
FWIW I don't like having a platform unsupported warning at the top of docs, it should be at bottom.

Reply:
It's not at the top. I only moved it above the asyncio REPL section because otherwise with the heading it suggested that the asyncio REPL is unavailable on WASM.